src/rules/containers.ts

```diff
@@ -1116,6 +1116,72 @@ components:
   },
 };
 
+export const containerEnvvarsStaticvalDeprecated: YAMLRule = {
+  name: "prop-component-container-envvars-staticval-deprecated",
+  type: "error",
```

Comment:
This is going to break everybody's yaml in the release editor -- what do you think of making this a `warn` so we don't block saving?

Reply:
oops. warn was the intention